docs/user-guide/map-views.md

```diff
@@ -0,0 +1,64 @@
+# Map Views
+
+The **Map Views** is a [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) tool useful to configure multiple 3D views and switch between them with navigation tool.
+
+## Add new view
+
+Once the user opens a map, in [3D Mode](navigation-toolbar.md#3d-navigation), the *Map Views* tools can be open through the <img src="../img/button/map-views-button.jpg" class="ms-docbutton" /> button available in the [Side Toolbar](mapstore-toolbars.md#side-toolbar).
+
+<img src="../img/map-views/map-views-panel.jpg" class="ms-docimage"/>
+
+To create a new view, the user can, using the compass located in the upper right corner of the map, highlight a specific area on the map, enable the layers, on TOC, to be displayed in the map and finally click on the <img src="../img/button/+++.jpg" class="ms-docbutton" />. This view will be visible in the *Views* list by clicking the <img src="../img/button/timeline-layers-list-button.jpg" class="ms-docbutton" /> button.
+
+<img src="../img/map-views/add-view.gif" class="ms-docimage"/>
+
+[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to customize and edit the new view by opening the Edit panel through the <img src="../img/button/editing-button.jpg" class="ms-docbutton" /> button. Here the user is allowed to:
+
+* Add a text, images, videos or hyperlinks by clicking **Description** section. The *description* is visible during the [Views Navigation](map-views.md#3d-views-navigations). Take a look at the following example.
+
+<img src="../img/map-views/description.jpg" class="ms-docimage"/>
+
+* Change the longitude, latitude and height of the *Camera Position* and *Center Position* by clicking on **Position** section.
+
+<img src="../img/map-views/position.jpg" class="ms-docimage"/>
+
+* Modify the duration of the animation and enable/disable the transition during the [Views Navigation](map-views.md#3d-views-navigations) by clicking on **Animation** section.
+
+<img src="../img/map-views/animation.jpg" class="ms-docimage"/>
+
+* Select a WFS layer to create a mask of a 3D Tile layer by clicking **Mask** section. The WFS layer is added to the map by the user and then used in a transparent way to create the clipping area.
+
+<img src="../img/map-views/mask-panel.jpg" class="ms-docimage"/>
+
+If the user enables the **Inverse** option, a hole is created inside the 3D Tile layers. This hole can be filled by other 3D tiles or 3D objects.
+
+<img src="../img/map-views/mask-inverse.jpg" class="ms-docimage"/>
+
+If the user disables the **Inverse** option the clipping style is used for the view, then the selected WFS layer masks the 3D Tile layers outside its polygon.
+
+<img src="../img/map-views/mask.jpg" class="ms-docimage"/>
+
+!!! note
+    The WFS layer can be only polygon feature and concave polygon are not supported by this type of clipping, only convex layers can be used from this purpose.
```

Comment:
This should refer to clipping inside the layer and not the mask option. So need to be moved below or merged with the clipping description of the layer